packages/venia-ui/lib/components/ProductOptions/options.js

```diff
@@ -34,6 +35,7 @@ class Options extends Component {
                 {...option}
                 key={option.attribute_id}
                 onSelectionChange={handleSelectionChange}
+                selectedVariant={selectedVariant}
```

Comment:
It's excessive to pass the entire `selectedVariant` object down to every single option block.

`selectedVariant` is a big object containing lots of props but the important one is `selectedVariant.options` an array of objects where the each object has a `label` and `value`. You could probably do a mapping here to pull off the specific selection that each `Option` cares about rather than asking the component to [figure it out](https://github.com/magento/pwa-studio/pull/1592/files#diff-5eee7b86d923aa851bd87db4f9e8b296R44).

[![Image from Gyazo](https://i.gyazo.com/423f59d6325249c41b10da7009a29fe0.png)](https://gyazo.com/423f59d6325249c41b10da7009a29fe0)